### PR TITLE
Add io.github.wxmaxima_developers.wxMaxima

### PR DESCRIPTION
**Pull request by bztd**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/wxMaxima-developers/wxmaxima/discussions/1835
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: io.gpl.wxmaxima.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+app-id: io.gpl.wxmaxima
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:1]: `gpl.io` seems to just be a parked domain. Is it under the control of the upstream developer of that project?

File: io.gpl.wxmaxima.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+modules:
+  - name: gnuplot
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=/app/ --with-qt=no
+      - make
+      - make install
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/gnuplot-5.4.10.tar.gz
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:15]: why isn't this downloaded fro gnuplot official downloads?

File: io.gpl.wxmaxima.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
+app-id: io.gpl.wxmaxima
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:3]: current runtime is 23.08. Please use that.

File: io.gpl.wxmaxima.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+sdk: org.freedesktop.Sdk
+command: wxmaxima
+modules:
+  - name: gnuplot
+    buildsystem: simple
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:8]: this should just work with `autotools`.

File: io.gpl.wxmaxima.yml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      - mkdir bb
+      - cd bb && ../configure --with-gtk --enable-webviewwebkit --prefix=/app/
+      - cd bb && make
+      - cd bb && make install
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/wxWidgets-3.2.2.1.tar.bz2
+        sha256: dffcb6be71296fff4b7f8840eb1b510178f57aa2eb236b20da41182009242c02
+  - name: wxmaxima
+    buildsystem: simple
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:47]: this should just work with `cmake-ninja` or `cmake`

File: io.gpl.wxmaxima.yml [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/wxmaxima-Version-23.10.0.tar.gz
+        sha256: 1c41352be0d88dc3b2413bab06f0a2e791579d61ac1e3a406c3349f5ea0fffe7
+  - name: desktop
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - install -Dm0644 wxMaxima128.png /app/share/icons/hicolor/128x128/apps/io.gpl.wxmaxima.png
+      - install -Dm0644 io.gpl.wxmaxima.desktop /app/share/applications/io.gpl.wxmaxima.desktop
+    sources:
+      - type: dir
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:65]: dont use `type: dir`.

File: io.gpl.wxmaxima.yml [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+        path: ./desktop
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:66]: there is no such path anyway.

File: io.gpl.wxmaxima.yml [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+finish-args:
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:67]: `finish-args` is usually found above modules.

File: io.gpl.wxmaxima.yml [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+      - --socket=x11
+      - --env=GNUPLOT_DRIVER_DIR=/app/libexec/gnuplot/5.4/
+      - --filesystem=host
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:70]: this is a very broad permission.

File: io.gpl.wxmaxima.yml [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - install -Dm0644 wxMaxima128.png /app/share/icons/hicolor/128x128/apps/io.gpl.wxmaxima.png
+      - install -Dm0644 io.gpl.wxmaxima.desktop /app/share/applications/io.gpl.wxmaxima.desktop
+    sources:
+      - type: dir
+        path: ./desktop
+finish-args:
+      - --socket=x11
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:68]: x11 requires `share=ipc` as well.

File: io.gpl.wxmaxima.yml [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - mkdir bb
+      - cd bb && ../configure --with-gtk --enable-webviewwebkit --prefix=/app/
+      - cd bb && make
+      - cd bb && make install
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/wxWidgets-3.2.2.1.tar.bz2
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:44]: why isn't this downloaded from wxWidgets?

File: io.gpl.wxmaxima.yml [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/gnuplot-5.4.10.tar.gz
+        sha256: 975d8c1cc2c41c7cedc4e323aff035d977feb9a97f0296dd2a8a66d197a5b27c
+  - name: maxima
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - cp -r ./sbcl/* /app/
+      - mkdir bbbb
+      - cd bbbb && ../configure --enable-sbcl --prefix=/app/
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:23]: why not `buildsystem: autotools`

File: io.gpl.wxmaxima.yml [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/wxmaxima-Version-23.10.0.tar.gz
+        sha256: 1c41352be0d88dc3b2413bab06f0a2e791579d61ac1e3a406c3349f5ea0fffe7
+  - name: desktop
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - install -Dm0644 wxMaxima128.png /app/share/icons/hicolor/128x128/apps/io.gpl.wxmaxima.png
+      - install -Dm0644 io.gpl.wxmaxima.desktop /app/share/applications/io.gpl.wxmaxima.desktop
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:63]: I'll guess that you are also probably missing the AppStream file.

File: io.gpl.wxmaxima.yml [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - mkdir build-wxm
+      - cd build-wxm && cmake -DCMAKE_INSTALL_PREFIX=/app/ -DwxWidgets_INCLUDE_DIRS=/app/include/wx-3.2 -DwxWidgets_LIBRARIES=app/lib -DWXM_DISABLE_WEBVIEW=on ../
+      - cd build-wxm && cmake --build .
+      - cd build-wxm && cmake --build . -- install
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/wxmaxima-Version-23.10.0.tar.gz
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:56]: I'm not sure what it is with binaries checked into git, but:

1. it doesn't look right because this is not how git is used
2. you reference master, which is wrong since as soon as the repo changes things might break.
3. See my comment about not using upstream downloads for gnuplot and wxWidget, for example... (and like other packages)

File: io.gpl.wxmaxima.yml (hunk 1 above)
Comment (hfiguiere) [reply io.gpl.wxmaxima.yml:1]: According to this 
https://github.com/wxMaxima-developers/wxmaxima/blob/Version-23.10.0/data/io.github.wxmaxima_developers.wxMaxima.desktop

this is just wrong.

Comment (bztd) [reply io.gpl.wxmaxima.yml:1]: I do not have control of the wxmaxima domain so I chose an arbitrary id. Should I change the id to io.github.wxmaxima_developers.wxMaxima? Do I have to do a new pull request?

File: io.gpl.wxmaxima.yml (hunk 14 above)
Comment (bztd) [reply io.gpl.wxmaxima.yml:56]: I use the repository because it was easier for me when creating the manifest and gives me greater control over the project. Regarding stability, I was planning to use the repository only to store the sources of the final versions. I suppose it's mandatory to use the original sources, right ?

File: io.gpl.wxmaxima.yml (hunk 1 above)
Comment (hfiguiere) [reply io.gpl.wxmaxima.yml:1]: > so I chose an arbitrary id.

Definitely not.

Use upstream. It's all setup for that it seems.

> Do I have to do a new pull request?

Why?

File: io.gpl.wxmaxima.yml (hunk 14 above)
Comment (hfiguiere) [reply io.gpl.wxmaxima.yml:56]: Use upstream sources.

File: io.gpl.wxmaxima.yml (hunk 13 above)
Comment (AsciiWolf) [reply io.gpl.wxmaxima.yml:63]: It looks like that upstream contains AppStream metadata file: https://github.com/wxMaxima-developers/wxmaxima/blob/main/data/io.github.wxmaxima_developers.wxMaxima.appdata.xml

It also contains desktop and icon files: https://github.com/wxMaxima-developers/wxmaxima/tree/main/data

Comment (hfiguiere) [reply io.gpl.wxmaxima.yml:63]: yes, see https://github.com/flathub/flathub/pull/4641#discussion_r1375337636

File: io.gpl.wxmaxima.yml [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+      - --socket=x11
+      - --env=GNUPLOT_DRIVER_DIR=/app/libexec/gnuplot/5.4/
+      - --filesystem=host:rw
+      
+modules:
+  - name: gnuplot
+    buildsystem: autotools
+    no-autogen: true
+    config-opts:
+      - --prefix=/app/
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:18]:
```suggestion
```

this is done automatically

File: io.gpl.wxmaxima.yml [hunk 16]
```diff
@@ -0,0 +1,76 @@
...
+      - cp -r ./sbcl/* /app/
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/f/sbcl-2.3.9.tar.gz
+        sha256: 4b685a3b834d1faf0976e936750062e4c8ad5718daf115075615402913b69121
+  - name: maxima
+    buildsystem: autotools
+    no-autogen: true
+    config-opts:
+      - --prefix=/app/
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:37]:
```suggestion
```
this is done automatically

File: io.gpl.wxmaxima.yml [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+      - --enable-sbcl
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/maxima/Maxima-source/5.47.0-source/maxima-5.47.0.tar.gz
+        sha256: 9104021b24fd53e8c03a983509cb42e937a925e8c0c85c335d7709a14fd40f7a
+  - name: wxwidgets
+    buildsystem: autotools
+    no-autogen: true
+    config-opts:
+      - --prefix=/app/
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:47]:
```suggestion
```

this is done automatically

File: io.gpl.wxmaxima.yml [hunk 18]
```diff
@@ -0,0 +1,76 @@
...
+      - --enable-webviewwebkit
+    sources:
+      - type: archive
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.2.3/wxWidgets-3.2.3.tar.bz2
+        sha256: c170ab67c7e167387162276aea84e055ee58424486404bba692c401730d1a67a
+  - name: wxmaxima
+    buildsystem: cmake-ninja
+    no-autogen: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app/
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:58]:
```suggestion
```

this is done automatically

File: io.gpl.wxmaxima.yml [hunk 19]
```diff
@@ -0,0 +1,76 @@
...
+      - type: archive
+        url: https://github.com/wxMaxima-developers/wxmaxima/archive/refs/tags/Version-23.10.0.tar.gz
+        sha256: 1c41352be0d88dc3b2413bab06f0a2e791579d61ac1e3a406c3349f5ea0fffe7
+  - name: desktop
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - install -Dm0644 wxMaxima128.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm0644 ${FLATPAK_ID}.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm0644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:72]: these files are already in the upstream repository. there is absolutely no reason to use use different ones.

File: io.gpl.wxmaxima.yml [hunk 20]
```diff
@@ -0,0 +1,76 @@
...
+      - --prefix=/app/
+      - --with-gtk
+      - --enable-webviewwebkit
+    sources:
+      - type: archive
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.2.3/wxWidgets-3.2.3.tar.bz2
+        sha256: c170ab67c7e167387162276aea84e055ee58424486404bba692c401730d1a67a
+  - name: wxmaxima
+    buildsystem: cmake-ninja
+    no-autogen: true
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:56]:
```suggestion
```

it's cmake.

File: io.gpl.wxmaxima.yml [hunk 21]
```diff
@@ -0,0 +1,76 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: wxmaxima
+
+finish-args:
+      - --share=ipc
+      - --socket=x11
+      - --env=GNUPLOT_DRIVER_DIR=/app/libexec/gnuplot/5.4/
+      - --filesystem=host:rw
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:11]: this is a very broad filesystem permission.

File: io.gpl.wxmaxima.yml [hunk 22]
```diff
@@ -0,0 +1,76 @@
...
+      - type: archive
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.2.3/wxWidgets-3.2.3.tar.bz2
+        sha256: c170ab67c7e167387162276aea84e055ee58424486404bba692c401730d1a67a
+  - name: wxmaxima
+    buildsystem: cmake-ninja
+    no-autogen: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app/
+      - -DwxWidgets_INCLUDE_DIRS=/app/include/wx-3.2 
+      - -DwxWidgets_LIBRARIES=app/lib
```
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:60]:
```suggestion
      - -DwxWidgets_LIBRARIES=/app/lib
```

I'm pretty sure you are missing the `/`

File: io.gpl.wxmaxima.yml
Comment (hfiguiere) [inline io.gpl.wxmaxima.yml:1]: the manifest file name must match the app-id

File: io.github.wxmaxima_developers.wxmaxima.yml [hunk 23]
```diff
@@ -0,0 +1,71 @@
...
+      - -DwxWidgets_LIBRARIES=/app/lib
+      - -DWXM_DISABLE_WEBVIEW=on
+    sources:
+      - type: archive
+        url: https://github.com/wxMaxima-developers/wxmaxima/archive/refs/tags/Version-23.10.0.tar.gz
+        sha256: 1c41352be0d88dc3b2413bab06f0a2e791579d61ac1e3a406c3349f5ea0fffe7
+  - name: desktop
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
```
Comment (hfiguiere) [inline io.github.wxmaxima_developers.wxmaxima.yml:64]:
```suggestion
    post-install:
```

File: io.github.wxmaxima_developers.wxmaxima.yml [hunk 24]
```diff
@@ -0,0 +1,71 @@
...
+      - install -Dm0644 data/${FLATPAK_ID}.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm0644 data/${FLATPAK_ID}.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm0644 data/${FLATPAK_ID}.appdata.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
```
Comment (hfiguiere) [inline io.github.wxmaxima_developers.wxmaxima.yml:67]: actually do they really need to be installed here?

File: io.github.wxmaxima_developers.wxmaxima.yml [hunk 25]
```diff
@@ -0,0 +1,71 @@
...
+    sources:
+      - type: file
```
Comment (hfiguiere) [inline io.github.wxmaxima_developers.wxmaxima.yml:69]:
```suggestion
      - type: archive
```

File: io.github.wxmaxima_developers.wxmaxima.yml (hunk 24 above)
Comment (bztd) [reply io.github.wxmaxima_developers.wxmaxima.yml:67]: that is the default location

Comment (hfiguiere) [reply io.github.wxmaxima_developers.wxmaxima.yml:67]: this wasn't the question.

Comment (hfiguiere) [reply io.github.wxmaxima_developers.wxmaxima.yml:67]: the question is do you need to install them. "here" mean "here in the flatpak build".

Comment (bztd) [reply io.github.wxmaxima_developers.wxmaxima.yml:67]: I misunderstood, I thought you had to install these files manually to display the application in the menu. So there is no need to do it?

File: io.gpl.wxmaxima.yml (hunk 9 above)
Comment (bztd) [reply io.gpl.wxmaxima.yml]: I think that limiting access to the file system harms the user experience.

File: io.github.wxmaxima_developers.wxmaxima.yml (hunk 23 above)
Comment (bztd) [reply io.github.wxmaxima_developers.wxmaxima.yml:64]: Why would I do this?

**Comment by hfiguiere** [comment]
bot, build io.github.wxmaxima_developers.wxmaxima

File: io.github.wxmaxima_developers.wxmaxima.yml (hunk 24 above)
Comment (hfiguiere) [reply io.github.wxmaxima_developers.wxmaxima.yml:67]:
```
install: cannot stat 'data/io.github.wxmaxima_developers.wxmaxima.png': No such file or directory
```

Comment (AsciiWolf) [reply io.github.wxmaxima_developers.wxmaxima.yml:67]: That's because the upstream name seems to be _io.github.wxmaxima_developers.wxMaxima_ (upper case Maxima).

File: io.github.wxmaxima_developers.wxMaxima.yml [hunk 26]
```diff
@@ -0,0 +1,64 @@
...
+        url: https://prdownloads.sourceforge.net/gnuplot/5.4.10/gnuplot-5.4.10.tar.gz
+        sha256: 975d8c1cc2c41c7cedc4e323aff035d977feb9a97f0296dd2a8a66d197a5b27c
+  - name: sbcl
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - cp -r ./sbcl/* /app/
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/f/sbcl-2.3.9.tar.gz
```
Comment (hfiguiere) [inline io.github.wxmaxima_developers.wxMaxima.yml:30]: what about using the official download too?
http://www.sbcl.org/platform-table.html

File: io.github.wxmaxima_developers.wxMaxima.yml [hunk 27]
```diff
@@ -0,0 +1,64 @@
...
+  - name: wxmaxima
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DwxWidgets_INCLUDE_DIRS=/app/include/wx-3.2 
+      - -DwxWidgets_LIBRARIES=/app/lib
+      - -DWXM_DISABLE_WEBVIEW=on
+    post-install:
+      - install -Dm0644 data/${FLATPAK_ID}.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm0644 data/${FLATPAK_ID}.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm0644 data/${FLATPAK_ID}.appdata.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
```
Comment (hfiguiere) [inline io.github.wxmaxima_developers.wxMaxima.yml:60]: I'll ask again why you install these? it's supposed to be taken care off by the build system.

File: io.github.wxmaxima_developers.wxMaxima.yml [hunk 28]
```diff
@@ -0,0 +1,64 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: wxmaxima
+
+finish-args:
+      - --share=ipc
+      - --socket=x11
+      - --env=GNUPLOT_DRIVER_DIR=/app/libexec/gnuplot/5.4/
+      - --filesystem=host:rw
```
Comment (hfiguiere) [inline io.github.wxmaxima_developers.wxMaxima.yml:11]: this is still a too broad file system permission.

**Comment by hfiguiere** [comment]
bot, build io.github.wxmaxima_developers.wxMaxima

File: io.github.wxmaxima_developers.wxMaxima.yml (hunk 26 above)
Comment (hfiguiere) [reply io.github.wxmaxima_developers.wxMaxima.yml:30]: And build from source, because:
```
/bin/sh: line 1: /app/bin/sbcl: cannot execute binary file: Exec format error
```

(on aarch64)

Comment (bztd) [reply io.github.wxmaxima_developers.wxMaxima.yml:30]: To compile sbcl you need sbcl. I'm looking for an alternative

Comment (hfiguiere) [reply io.github.wxmaxima_developers.wxMaxima.yml:30]: from the page linked above:

image

**Comment by bztd** [comment]
bot, build io.github.wxmaxima_developers.wxmaxima

**Comment by bztd** [comment]
bot, build io.github.wxmaxima_developers.wxMaxima

**Comment by bztd** [comment]
bot, build io.gpl.wxmaxima

**Comment by bztd** [comment]
bot, build io.github.wxmaxima_developers.wxMaxima

**Comment by bztd** [comment]
bot, build io.github.wxmaxima_developers.wxMaxima

**Comment by bztd** [comment]
bot, build io.github.wxmaxima_developers.wxMaxima

File: io.github.wxmaxima_developers.wxMaxima.yml [hunk 29]
```diff
@@ -0,0 +1,75 @@
+app-id: io.github.wxmaxima_developers.wxMaxima
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: wxmaxima
+finish-args:
+      - --share=ipc
+      - --socket=x11
+      - --env=LC_CTYPE="en_US.UTF-8"
```
Comment (AsciiWolf) [inline io.github.wxmaxima_developers.wxMaxima.yml:9]: Why is this needed?

File: io.github.wxmaxima_developers.wxMaxima.yml [hunk 30]
```diff
@@ -0,0 +1,75 @@
...
+      - --filesystem=home:rw
```
Comment (AsciiWolf) [inline io.github.wxmaxima_developers.wxMaxima.yml:10]: Is full user home dir access really required?

**Comment by AsciiWolf** [comment]
From the wxMaxima documentation:

> The frontends wxMaxima and Xmaxima communicate with the backend using a local network connection.

If I understand this properly, it is possible to use wxMaxima to connect to a backend not running on localhost. Maybe adding `--share=network` could be a good idea?

File: io.github.wxmaxima_developers.wxMaxima.yml (hunk 29 above)
Comment (bztd) [reply io.github.wxmaxima_developers.wxMaxima.yml:9]: As for the LC_CTYPE environment variable, it prevents a warning message from being displayed when executing a .mac file that prints to the screen.

File: io.github.wxmaxima_developers.wxMaxima.yml (hunk 30 above)
Comment (bztd) [reply io.github.wxmaxima_developers.wxMaxima.yml:10]: Access to the file system is necessary to load saved sessions or to load scripts, save numerical results or images. Although you can restrict access to a specific directory, this would detract from the user experience.

File: io.github.wxmaxima_developers.wxMaxima.yml (hunk 29 above)
Comment (hfiguiere) [reply io.github.wxmaxima_developers.wxMaxima.yml:9]: that also mean it will display numbers in a weird way for most of the world.

File: io.gpl.wxmaxima.yml (hunk 2 above)
Comment (bztd) [reply io.gpl.wxmaxima.yml]: Is there a way to pass the user's environment variable to wxMaxima?

File: io.github.wxmaxima_developers.wxMaxima.yml (hunk 29 above)
Comment (bztd) [reply io.github.wxmaxima_developers.wxMaxima.yml:9]: Is there a way to pass the user's environment variable to wxMaxima?

**Comment by bztd** [comment]
As for what is required by battalion, I contacted the developer and he agreed to upload the application to flathub.

**Comment by bztd** [comment]
bot, build io.github.wxmaxima_developers.wxMaxima

**Comment by bztd** [comment]
> > I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. Link:
> > I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
> 
> These two are marked as done. Can you elaborate?

I contacted the developer and he agreed to upload the application to flathub.
https://github.com/wxMaxima-developers/wxmaxima/discussions/1835
